src/main.cpp

```diff
@@ -8,6 +10,20 @@ int main(int argc, char *argv[])
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
     KiwixApp a(argc, argv);
 
+    // format systems language
+    QString defaultLocale = QLocale::system().name(); // e.g. "de_DE"
+    defaultLocale.truncate(defaultLocale.lastIndexOf('_')); // e.g. "de"
+    QLocale::setDefault(defaultLocale);
```

Comment:
Why do you change the default locale ?
We should keep the full locale, Qt already has a mechanism to fallback to `fr` if `fr_FR` doesn't exist (https://doc.qt.io/qt-5/qtranslator.html#load)

If we change the locale to `fr` we will not be able to have different translation for `fr_FR` and `fr_CA`

Reply:
Great